packages/jitar-vite-plugin/package.json

```diff
@@ -17,6 +17,9 @@
     "clean": "rm -rf dist build",
     "release": "npm run clean && npm run build && npm publish"
   },
+  "dependencies": {
+    "jitar-reflection": "^0.1.0"
```

Comment:
This dependency doesn't exist anymore since the version of it has been set to 0.3.3